src/applications/personalization/profile/constants.js

```diff
@@ -92,6 +92,28 @@ export const BANK_INFO_UPDATED_ALERT_SETTINGS = {
 
 export const NOTIFICATION_GROUPS = Object.freeze({
   APPLICATIONS: 'group1',
+  GENERAL: 'group2',
```

Comment:
Confirmed that this is the 'new' group that will show up in the API response for general updates